lib/serializer.js

```diff
@@ -77,11 +78,18 @@ module.exports = class CustomSerializer extends Serializer {
 
     _serializeSlot(node) {
         const slotName = node.attribs.name;
-        if (this.slotMap.has(slotName)) {
+        if (slotName && this.slotMap.has(slotName)) {
             const slots = this.slotMap.get(slotName);
             if (slots) {
                 slots.forEach(this._serializeNode);
             }
+        } else {
+            // Unnamed slot behaviour
+            const defaultSlots = this.slotMap.get('default');
+            this.defaultSlotsInserted = true;
```

Comment:
We should check what does the spec say about duplicate default slots. Maybe we should also warn on that.